src/app/Archives/AllArchivedRecordingsTable.tsx

```diff
@@ -168,12 +168,29 @@ export const AllArchivedRecordingsTable: React.FC<AllArchivedRecordingsTableProp
   }, [context.settings, refreshDirectoriesAndCounts]);
 
   React.useEffect(() => {
-    addSubscription(
-      context.notificationChannel.messages(NotificationCategory.RecordingMetadataUpdated).subscribe(() => {
-        refreshDirectoriesAndCounts();
-      }),
-    );
-  }, [addSubscription, context.notificationChannel, refreshDirectoriesAndCounts]);
+    const subscription = context.notificationChannel.messages(NotificationCategory.RecordingMetadataUpdated).subscribe(event => {
```

Comment:
Is there a reason to stop using `addSubscription` here? If so, it should be removed from the hook dependency array.